dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/README.md

```diff
@@ -92,6 +92,92 @@ const decimal shippingCost = 2.5m;
 var totalPrice = price + shippingCost;
 ```
 
+This also applies to switch statements:
+
+Code with violation:
+```csharp
+ var check = 11;
+
+  switch (11):
+ {
+ case check:
+     return;
+ default:
+     return;
+ } 
+```
+
+Code with fix:
+```csharp
+var check = 11;
+
+ switch (check):
+{
+case check:
+    return;
+default:
+    return;
+} 
+```
+
+for loops:
+Code with violation:
+```csharp
+for(int testVar = 7; i > 0; i--)
+    {
+        continue;
+    }
+```
+Code with fix:
+```csharp
+var check = 11;
+var iterator = 2;
+
+ for (var counter = iterator; counter < check; counter++)
+{
+
+}
+```
+
+while loops:
+Code with violation:
+```csharp
+while(count < 11)
+    {
+        count++;
+    }
+```
+Code with fix:
+```csharp
+var check = 11;
+var toCheck = 0;
+ while(toCheck < check)
```

Comment:
quite a lot of indentation changes here, are you OK to review?
![image](https://github.com/user-attachments/assets/7a4c4426-c20f-48b0-a71a-ea4b63d52644)